app/src/renderer/scripts/boot.js

```diff
@@ -109,8 +109,12 @@ export const startApp = async (
     store.commit(`setRpcUrl`, urlParams.rpc)
   }
 
-  store.dispatch(`connect`)
   store.dispatch(`showInitialScreen`)
+  store.dispatch(`connect`)
+    // wait for connected as the check for session will sign in directly and query account data
+    .then(() => {
```

Comment:
why not `await` ?

Reply:
because then it would await until connected, but we want to return the vue instance on line 119